torch/csrc/distributed/c10d/NCCLUtils.cpp

```diff
@@ -11,16 +11,15 @@ std::string getNcclVersion() {
   static std::string versionString;
 
   std::call_once(ncclGetVersionFlag, []() {
-    int version;
-    ncclResult_t status = ncclGetVersion(&version);
-    // can't compute the version if call did not return successfully or version
-    // code < 100 (corresponding to 0.1.0)
-    if (status != ncclSuccess || version < 100) {
+    int version = torch::cuda::nccl::version();
```

Comment:
do you need to include the following header before using this API? 

```c++
#include <torch/csrc/cuda/nccl.h>
```

The header does not seem to include that file:

https://github.com/pytorch/pytorch/blob/79203f6cab65ce908e54d183cbba65b8d44bf727/torch/csrc/distributed/c10d/NCCLUtils.hpp#L5-L12

IIUC, the `nccl.h` is the 3rd-party nccl not `cuda/nccl`.

Reply:
Added the header.

Did the following to get a clean build:
$ python setup.py clean
$ rm -rf build
$ python setup.py install

Then test:
$ python3 test/distributed/test_c10d_nccl.py       
ssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssss
----------------------------------------------------------------------
Ran 90 tests in 0.002s

OK (skipped=90)